release.sh

```diff
@@ -9,7 +9,9 @@ for v in $VERSIONS; do
   python test_resources.py
   python setup.py clean --all
   python setup.py bdist_wheel
+  python setup.py sdist_wheel
   python -m mkdocs build -d site/$MAIN_VERSION
   #twine upload dist/lightkube_models-${v}.*-py3-none-any.whl -r $1
   ls dist/lightkube_models-${v}.*-py3-none-any.whl
+  ls dist/lightkube_models-${v}.tar.gz
```

Comment:
```suggestion
  ls dist/lightkube_models-${v}.*.tar.gz
```

the generator version is added to the module version,  so we need to be sure to account for that.